libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java

```diff
@@ -151,7 +151,7 @@ private PendingIntent createPendingOpenIntent(Context context) {
 
   private void registerReceiver(Context context) {
     if (context != null) {
-      context.registerReceiver(endNavigationBtnReceiver, new IntentFilter(END_NAVIGATION_ACTION));
+      context.registerReceiver(endNavigationBtnReceiver, new IntentFilter(END_NAVIGATION_ACTION), Context.RECEIVER_NOT_EXPORTED);
```

Comment:
May bad, I seeing this too late. This call of [`registerReceiver`](https://developer.android.com/reference/android/content/Context#registerReceiver(android.content.BroadcastReceiver,%20android.content.IntentFilter,%20int)) is only available for API +26.

Reply:
I'm currently investigate some time in #94. And will fix this too on this way.

Reply:
Thanks for investigating @Fabi755